bld010/frontend-palette-picker
# App setup

#### What's this PR do?

Sets up general app structure, displays and locks random colors 

#### Where should the reviewer start?

ColorDisplay.js

#### How should this be manually tested?

Run local server and verify that random colors are generated and you can lock a color

#### Any background context you want to provide?
#### What are the relevant tickets?

[Lock colors](https://app.clubhouse.io/briannadelvalle/story/50/as-a-user-i-want-to-be-able-to-lock-any-colors-i-see-and-generate-new-palettes-including-them-so-i-can-save-colors-i-like)

[Random color generation](https://app.clubhouse.io/briannadelvalle/story/49/as-a-user-i-want-to-be-able-to-see-randomly-generated-color-palettes-to-help-me-choose-palettes-for-designs)

#### Screenshots (if appropriate)
#### Questions:
